tasks/eth/ink-001-permissionless-proofs/VALIDATION.md

```diff
@@ -0,0 +1,113 @@
+
+- **Key**: `0xe34b8b74e1cdcaa1b90aa77af7dd89e496ad9a4ae4a4d4759712101c7da2dce6` <br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After**: `0x000000000000000000000000000000000000000000000000011c37937e080000` <br/>
+  **Meanning**: Updates the `PermissionedDisputeGame` initial bond amount to 0.08 ETH. Verify that the slot is correct using `cast index uint 1 102`. Where `1` is the game type and 102 is the [storage slot](https://github.com/ethereum-optimism/optimism/blob/33f06d2d5e4034125df02264a5ffe84571bd0359/packages/contracts-bedrock/snapshots/storageLayout/DisputeGameFactory.json#L48).
+
+- **Key**: `0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b` <br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After**: `0x0000000000000000000000006A8eFcba5642EB15D743CBB29545BdC44D5Ad8cD` <br/>
+  **Meaning**: Updates the implementation for game type 0. Verify that the slot is correct using `cast index uint 0 101` where 0 is the game type and 101 is the [storage slot](https://github.com/ethereum-optimism/optimism/blob/33f06d2d5e4034125df02264a5ffe84571bd0359/packages/contracts-bedrock/snapshots/storageLayout/DisputeGameFactory.json#L41) of the `gameImpls` mapping.
+
+### `0xde744491BcF6b2DD2F32146364Ea1487D75E2509` (`AnchorStateRegistryProxy`)
+
+- **Key**: `0xa6eef7e35abe7026729641147f7915573c7e97b47efa546f5f6e3230263bcb49`<br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` (Note this may have changed if games of this type resolved)<br/>
+  **After**: `0x5220f9c5ebf08e84847d542576a67a3077b6fa496235d93c557d5bd5286b431a` <br/>
+  **Meaning**: Set the anchor state output root for game type 0 to 0x5220f9c5ebf08e84847d542576a67a3077b6fa496235d93c557d5bd5286b431a. This is the slot for the `anchors` mapping, which can be computed as `cast index uint 0 1`, where 0 is the game type and 1 is the slot of the `anchors` mapping. If you have access to the Ink Mainnet's op-node RPC endpoint and want to verify the new root, you can use the following command to verify:
+  ```
+  cast rpc --rpc-url $OP_NODE_RPC "optimism_outputAtBlock" $(cast th 523052)
+  ```
+
+- **Key**: `0xa6eef7e35abe7026729641147f7915573c7e97b47efa546f5f6e3230263bcb4a`<br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` (Note this may have changed if games of this type resolved)<br/>
+  **After**: `0x000000000000000000000000000000000000000000000000000000000007fb2c`<br/>
+  **Meaning**: Set the anchor state L2 block number for game type 0 to 523052. The slot number can be calculated using the same approach as above, and incremented by 1, based on the contract storage layout
+
+### Safe Contract State Changes
+
+The only other state changes should be restricted to one of the following addresses:
+
+- L1 2/2 ProxyAdmin Owner Safe: `0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A`
```

Comment:
The nonce in the validation are incorrect. 
This should respect from our notion doc and the L1PAO should increase from 7 -> 8 because the task 022 is executed before. 
The SC's nonce should be increased from 9 -> 10 as the task 022 is executed before. 
The FuS should be increased from the 12 -> 13 as also the task 021 and 022 will increase two times the nonce before.

Reply:
updated!

Reply:
Thanks!